cmd/ctr/commands/images/pull.go

```diff
@@ -86,6 +90,12 @@ command. As part of this process, we do the following:
 			return fmt.Errorf("please provide an image reference to pull")
 		}
 
+		if context.Bool("print-chainid") {
+			if context.Bool("no-unpack") {
+				return fmt.Errorf("--print-chainid and --no-unpack are incompatible options")
```

Comment:
Here it would be more correct to use ```errors.New```.

Reply:
I'll update this; was just a copy-paste from `ctr images import`:

https://github.com/containerd/containerd/blob/main/cmd/ctr/commands/images/import.go#L207